vowpalwabbit/cbify.cc

```diff
@@ -126,6 +97,14 @@ void copy_example_to_adf(cbify& data, example& ec)
   }
 }
 
+vector<float> a_s_get_scores(ACTION_SCORE::action_scores& a_s)
+{
+  vector<float> scores(a_s.size());
```

Comment:
Can we shift the allocation into cb_explore_adf?

Reply:
done

Reply:
Does this function need to be cleaned up?

Reply:
done

Reply:
done